index.html

```diff
@@ -0,0 +1,226 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header id="header">
+        <h3 id="logo" class="content-padding"><a href="#intro">My Logo</a></h3>
+        <nav id="navbar">
+            <ul>
+                <li class="content-padding web"><a href="#works">Portfolio</a></li>
+                <li class="content-padding web"><a href="#about-myself">About</a></li>
+                <li class="content-padding web"><a href="#contact-form">Contact</a></li>
+                <li class="content-padding mobile"><a href="#"><img src="images/menu-icon.png" alt="menu"></a></li>
+            </ul>
+        </nav>
+        <div class="mobile-menu">
+            <img src="images/cross.svg" alt="cross">
+            <ul>
+                <li class=""><a href="#works">Portfolio</a></li>
+                <li class=""><a href="#about-myself">About</a></li>
+                <li class=""><a href="#contact-form">Contact</a></li>
+            </ul>
+        </div>
+    </header>
```

Comment:
- [ ] Kindly notice that you should give the HTML elements the `ids` and the `classes` names so the javascript code can work properly and the mobile menu can open and close properly.

```html
<header id="header">
        <h3 id="logo" class="content-padding"><a href="#intro">My Logo</a></h3>
        <nav id="navbar">
            <ul>
                <li class="content-padding web"><a href="#works">Portfolio</a></li>
                <li class="content-padding web"><a href="#about-myself">About</a></li>
                <li class="content-padding web"><a href="#contact-form">Contact</a></li>
                <li class="content-padding mobile"><a href="#" id="navbar-hamburger"><img src="images/menu-icon.png" alt="menu"></a></li>
            </ul>
        </nav>
        <div class="mobile-menu">
            <img id="cross" src="images/cross.svg" alt="cross">
            <ul>
                <li class="nav-link"><a href="#works">Portfolio</a></li>
                <li class="nav-link"><a href="#about-myself">About</a></li>
                <li class="nav-link"><a href="#contact-form">Contact</a></li>
            </ul>
        </div>
</header>
```